### PR TITLE
Closes #692, adds 0.0.0.0 and 127.0.0.1 to development settings

### LINKED ISSUE #692: Default site URL announced by docker should not be trusted
Docker on start announces `Starting development server at http://0.0.0.0:8000/`. It is wrong and it is available under  http://127.0.0.1:8000/. This should be reflected somewhere in documentation.

### PR DESCRIPTION
**Pull request by Hiyorimi**

### CODE REVIEW

File: {{cookiecutter.project_name}}/server/settings/environments/development.py [hunk 1]
```diff
@@ -15,6 +15,11 @@
 
 DEBUG = True
 
+ALLOWED_HOSTS = [
+    config('DOMAIN_NAME'),
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/server/settings/environments/development.py]: Is there a way to preserve a `django`'s default? Something like:

```python
ALLOWED_HOSTS = [
    *default_settings.ALLOWED_HOSTS,
    config('DOMAIN_NAME'),
    '0.0.0.0',
]
```

Comment (Hiyorimi) [reply {{cookiecutter.project_name}}/server/settings/environments/development.py]: I can just add `['localhost', '127.0.0.1', '[::1]']` from [this](https://docs.djangoproject.com/en/2.1/ref/settings/) documentation page.

Comment (Hiyorimi) [reply {{cookiecutter.project_name}}/server/settings/environments/development.py]: @sobolevn any news?

Comment (sobolevn) [reply {{cookiecutter.project_name}}/server/settings/environments/development.py]: Agree

**Comment by Hiyorimi** [comment]
Updated

**Comment by sobolevn** [comment]
@Hiyorimi please, fix the build
> server/settings/environments/development.py:19: error: Name 'config' is not defined
> The command "sh tests/test_integration.sh" exited with 1.

**Comment by Hiyorimi** [comment]
fixed

**Comment by sobolevn** [comment]
Thanks!

**Pull request merged**